_pages/en/strategy-transform-high-performing-org.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: "Strategy to Transform IITB into a High Performing Organization (Continuous Improvement)"
+ref: transform-high-performing-org
+lang: en
+status: posted
+sections: Strategies
+permalink: /strategy-transform-high-performing-org.html
+---
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+  - [Governance, Compliance, and Reporting](#governance-compliance-and-reporting)
+  - [Finance](#finance)
+  - [Legal](#legal)
+  - [Architecture](#architecture)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+
+## Introduction
```

Comment:
In ihe Introduction or as a background brief we can mention that this strategy is not something new or is not something that will take a specific time of the day. Most of the teams are already doing it (improving) by simply fixing a bug, using Kanban boards,  attending meetings, taking training, attending conferences, webinars,  and sharing the knowledge, or just reading an article. It is just outlining a skill and a framework of continuously improving for yourself and for the team.  We are all already part of this daily routine without knowing it.

Reply:
"In ihe Introduction or as a background brief we can mention that this strategy is not something new or is not something that will take a specific time of the day."

We certainly can, and this is certainly true. However, the strategy cant be "we want people to spend more time like this." It needs to have actions and outcomes. So people spending part of their day towards improvement is true, but why? What do we expect to improve by doing this? How can we measure it? At the moment, it is difficult. So part of the strategy, I would argue, should be (as an action) to put in place the required tooling to ensure we can measure the expected outcomes from the strategy.

All that to stay, the "output" of the strategy is actions, like developing tooling and measuring outcomes. Any in order for the measuring to be possible, how work is done will need to change (which I think is what you are alluding to above)